recipes/perl-business-issn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set perl_name = "Business::ISSN" %}
+{% set name = "perl-" + perl_name.replace("::", "-").lower() %}
+{% set author = "BDFOY" %}
+{% set version = "1.005" %}
+{% set sha256 = "3b09b0267f0a6660fb92b6f50c4c7796ef6a263b62ad3bbeaa07189a0c7c39b3" %}
```

Comment:
Please see my comments in https://github.com/conda-forge/staged-recipes/pull/28602 about reducing Jinja usage.